src/withHelmet.js

```diff
@@ -0,0 +1,14 @@
+/* eslint react/display-name: 0 */
+import React from "react";
+import createElement from "recompose/createElement";
+import createHelper from "recompose/createHelper";
+import Helmet from "./Helmet.js";
+
+const withHelmet = (propsMapper, BaseComponent) => props => (
+  <div>
```

Comment:
@mull Helmet doesn't take children as it messes with the `shouldComponentUpdate` check.